src/CodeBlock.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace SilverStripe\MD_PHP_CodeSniffer;
+
+use PHP_CodeSniffer\Files\DummyFile;
+
+class CodeBlock extends DummyFile
+{
+    public int $num;
+
+    public string $realPath;
+
+    private string $finalContent = '';
```

Comment:
This property had a default value while the others don't, should pop in default values for each of them?

Reply:
The others don't need default values because they're explicitly being set when instantiating the thing. I'll make that clearer by having them passed into the constructor and instantiating them there instead.

Reply:
Done